yew-macro/src/html_tree/html_element.rs

```diff
@@ -1,22 +1,18 @@
-mod tag_attributes;
-
-use super::{
-    HtmlChildrenTree, HtmlDashedName, HtmlProp as TagAttribute, HtmlPropSuffix as TagSuffix,
-};
+use super::{HtmlChildrenTree, HtmlDashedName, TagTokens};
+use crate::props::{ClassesForm, ElementProps, Prop};
 use crate::stringify::Stringify;
 use crate::{non_capitalized_ascii, stringify, Peek, PeekValue};
 use boolinator::Boolinator;
-use proc_macro2::{Delimiter, Span, TokenStream};
+use proc_macro2::{Delimiter, TokenStream};
 use quote::{quote, quote_spanned, ToTokens};
 use syn::buffer::Cursor;
-use syn::parse::{Parse, ParseStream, Result as ParseResult};
+use syn::parse::{Parse, ParseStream};
 use syn::spanned::Spanned;
 use syn::{Block, Ident, Token};
-use tag_attributes::{ClassesForm, TagAttributes};
 
 pub struct HtmlTag {
     tag_name: TagName,
-    attributes: TagAttributes,
```

Comment:
sure. Btw, do you think naming the `key=value` syntax "Prop" is an appropriate name?
I think we should unify the nomenclature for these things and I wasn't sure whether to go with "Attribute" or "Prop".
I only went with "Prop" because that's what the previous structure was called.

Reply:
Yeah I think unifying with "prop" is appropriate. It will have to get a bit more explicit when we get into this issue: https://github.com/yewstack/yew/issues/1322